app/fetch_preconfig.js

```diff
@@ -36,12 +36,27 @@ Client4.doFetchWithResponse = async (url, options) => {
         };
     }
 
+    const customHeaders = LocalConfig.CustomRequestHeaders;
+    let requestOptions = Client4.getOptions(options);
+    if (customHeaders &&
```

Comment:
Maybe have this just be `customHeaders && Object.keys(customHeaders)` to save space?